ipam/ippool_manager_test.go

```diff
@@ -384,6 +384,12 @@ var _ = Describe("IPPool manager", func() {
 							Namespace: "myns",
 						},
 					},
+					Status: ipamv1.IPClaimStatus{
+						Address: &corev1.ObjectReference{
+							Name:      "abcpref-192-168-1-11",
+							Namespace: "myns",
+						},
+					},
```

Comment:
When IpAddress created, it will update ipClaimStatus firtst and then will proceed with ip address creation. Here somehow we did not include status in IpClaim which is causing an error. Not sure how it was passing before.

Reply:
Interesting, it shouldn't have passed if thats the case.

Reply:
It was actually due to status handling changes in  [CR v0,15.0](https://github.com/kubernetes-sigs/controller-runtime/pull/2259). Fixed now